includes/common/common-functions.php

```diff
@@ -303,6 +303,9 @@ function getRegisteredShopId(): string
 function getShipmentFiles($post_id)
 {
     $getOrderMetaData = get_post_meta($post_id, GET_META_SHIPMENT_TRACKING_KEY, true);
+    if (!$getOrderMetaData) {
+        return;
+    }
     $getOrderMeta     = json_decode($getOrderMetaData);
```

Comment:
Why not throw an error when this happens? 

```suggestion
    $getOrderMeta     = json_decode($getOrderMetaData, true, 512, THROW_ON_ERRORS);
```

Reply:
That will not solve the `Uncaught TypeError: json_decode() expects parameter 1 to be string, bool given`.
To be honest, I just copied the flow of the existing code in those functions, which has several return statements.